src/terminal/Functions.h

```diff
@@ -352,6 +352,7 @@ constexpr inline auto CAPTURE     = detail::CSI('>', 0, 2, std::nullopt, 't', VT
 
 constexpr inline auto DECSSDT     = detail::CSI(std::nullopt, 0, 1, '$', '~', VTType::VT320, "DECSSDT", "Select Status Display (Line) Type");
 constexpr inline auto DECSASD     = detail::CSI(std::nullopt, 0, 1, '$', '}', VTType::VT420, "DECSASD", "Select Active Status Display");
+constexpr inline auto DECPS 			= detail::CSI(std::nullopt, 3, 18, ',', '~', VTType::VT520, "DECPS", "Controls the sound frequency or notes");
```

Comment:
is `18` a documented limit? Otherwise we could use `ArgsMax`.

Reply:
FYI, jerch did some testing of DECPS on a real VT525, and it wasn't actually able to handle more than one note at a time. It's possible the multi-note capabilities were introduced in later firmware revisions though.

In any event, we decided it best to keep the multi-note functionality anyway, since it was at least documented in the manual. But I haven't limited it to 16 in Windows Terminal - just the existing parameter limit, which for us is 30 notes (32 - 2). I think MinTTY might be the same. Not sure about XTerm.js.

Nothing wrong with limiting it to 16 though.

Reply:
I think then sticking with the value `18` (`16 + 2`) in there is okay, as long as the limit isn't less than what the spec is saying.

Reply:
```suggestion
constexpr inline auto DECPS       = detail::CSI(std::nullopt, 3, 18, ',', '~', VTType::VT520, "DECPS", "Controls the sound frequency or notes");
```
(use spaces instead of tabs, btw, you should be able to configure your editor to evaluate the project root's .editorconfig file, that contains this information :) )